iothub/device/samples/PnpDeviceSamples/Thermostat/Parameter.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using CommandLine;
+
+namespace Thermostat
+{
+    /// <summary>
+    /// Parameters for the application supplied via command line arguments. It default to environment variables..
+    /// </summary>
+    internal class Parameters
+    {
+        [Option(
+            's',
+            "DeviceProvisioningType",
```

Comment:
DpsSecurityType?

Reply:
Updated to DeviceSecurityType -> in line with the env var name.